test/jdk/javax/swing/AbstractButton/5049549/bug5049549.java

```diff
@@ -38,19 +38,31 @@
 import javax.swing.JPanel;
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
 
 public class bug5049549 {
 
-    private static ImageIcon DE = new ImageIcon(bug5049549.class.getResource("DE1.gif"));
-    private static ImageIcon DI = new ImageIcon(bug5049549.class.getResource("DI1.gif"));
-    private static ImageIcon DS = new ImageIcon(bug5049549.class.getResource("DS1.gif"));
-    private static ImageIcon RO = new ImageIcon(bug5049549.class.getResource("RO1.gif"));
-    private static ImageIcon RS = new ImageIcon(bug5049549.class.getResource("RS1.gif"));
-    private static ImageIcon SE = new ImageIcon(bug5049549.class.getResource("SE1.gif"));
-    private static ImageIcon PR = new ImageIcon(bug5049549.class.getResource("PR1.gif"));
-
     private static Blocker blocker = new Blocker();
 
+
+    private static Icon generateImage(String str) {
+
+        BufferedImage img = new BufferedImage(40, 30,
+                BufferedImage.TYPE_INT_RGB);
```

Comment:
```suggestion
    private static Icon generateImage(String str) {
        BufferedImage img = new BufferedImage(40, 30,
                BufferedImage.TYPE_INT_RGB);
```
A blank line between a method declaration and its first statement doesn't make much sense, I'm for removing this blank line.